ctfo_server.h

```diff
@@ -101,15 +101,26 @@ struct CTFOHypermedia
 
 CURRENT_STRUCT(CTFOServerParams) {
   CURRENT_FIELD(api_port, int);
+  CURRENT_FIELD_DESCRIPTION(api_port, "Port to spawn CTFO API on.");
   CURRENT_FIELD(rest_port, int, 0);
```

Comment:
Zero default value -> `Optional<uint16_t>` with comment "equals to `api_port` if not set"?